backend/app/assets/stylesheets/spree/backend/globals/_dark.scss

```diff
@@ -0,0 +1,14 @@
+html.dark-mode {
```

Comment:
Thoughts on using `prefers-color-scheme` instead?

```css
@media (prefers-color-scheme: dark) {
  ...
}
```

Ref: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

Looks like you can also use the `color-scheme` property to force a scheme on a particular element: https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme

This means that it could follow the system by default and allow opt-in to force it to be either way via JS by setting that property on `<html>`.